data/problems/023_extract_random.md

```diff
@@ -10,9 +10,10 @@ tags: [ "list" ]
 
 ```ocaml
 # let rand_select list n =
+    Random.init 0;
     let rec extract acc n = function
       | [] -> raise Not_found
-      | h :: t -> if n = 0 then (h, acc @ t) else extract (h :: acc) (n - 1) t
+      | h :: t -> if n = 0 then (h, List.rev acc @ t) else extract (h :: acc) (n - 1) t
```

Comment:
Since the elements are extracted randomly, it doesn't make much sense to re-order the list (it's an additional operation, but we don't care about the order).
If the solution doesn't match the returned value of the function with `Random.init 0`, let's change the solution instead :)

Reply:
I removed the unnecessary reverse operations :)